docs/site.json

```
@@ -1,6 +1,6 @@
 {
   "baseUrl": "",
-  "titlePrefix": "",
+  "titlePrefix": "MyProduct",
   "titleSuffix": "AddressBook Level-3",
```

Should we discuss on the title prefix and suffix? I am not sure about this

---

Was previous just following as per the Wk7 instructions, but with more consideration and since we can deviate(?) let's do TAPro - <page title> without a suffix. Reason being most times suffix is used for "online" or "live".

---

The [MarkBind User Guide](https://markbind.org/userGuide/siteJsonFile.html#titleprefix) states that the prefix/suffix will appear on all page titles, so I think we should leave either `titlePrefix` or `titleSuffix` blank. In my opinion, I think we should do this:
```suggestion
  "titlePrefix": "",
  "titleSuffix": "TA Pro",
```